packages/suite/src/components/firmware/FirmwareInitial.tsx

```diff
@@ -160,16 +160,17 @@ export const FirmwareInitial = ({
         availableFwVersion === currentFwVersion
     );
     const bitcoinOnlyDevice = isBitcoinOnlyDevice(device);
-    const isCurrentlyBitcoinOnly = hasBitcoinOnlyFirmware(device) || bitcoinOnlyDevice;
+    const isCurrentlyBitcoinOnly = hasBitcoinOnlyFirmware(device);
+    const noFirmwareInstalled = ['none', 'unknown'].includes(device.firmware);
     const targetFirmwareType =
-        // switching to Regular
-        (isCurrentlyBitcoinOnly && shouldSwitchFirmwareType) ||
-        // updating Regular
-        (!isCurrentlyBitcoinOnly && !shouldSwitchFirmwareType) ||
-        // attempting to switch to Bitcoin-only from old firmware
-        (!isCurrentlyBitcoinOnly && shouldSwitchFirmwareType && !isBitcoinOnlyAvailable)
-            ? FirmwareType.Regular
-            : FirmwareType.BitcoinOnly;
+        // updating Bitcoin-only
+        (isCurrentlyBitcoinOnly && !shouldSwitchFirmwareType) ||
+        // switching to Bitcoin-only
+        (!isCurrentlyBitcoinOnly && shouldSwitchFirmwareType && isBitcoinOnlyAvailable) ||
+        // installing an empty Bitcoin-only device
```

Comment:
```suggestion
        // initialising an empty Bitcoin-only device
```

or installing fw on an empty ...

Reply:
Simplified further.